templates/graphql/playground.html

```diff
@@ -46,7 +46,11 @@
       GraphQLPlayground.init(document.getElementById('root'), {
         settings: {
           'schema.polling.enable': false
-        }
+        },
+        tabs: [{
+          endpoint: '{{ api_url }}',
+          query: `{{ query|safe }}`
+        }]
```

Comment:
Could we try to fix the issue where this would override the tabs from local storage? The feature of saving tabs is really nice and useful

Reply:
Good point. According to Playground's [readme](https://github.com/prisma-labs/graphql-playground) that's not possible:

> Note: When using this feature, tabs will be resetted each time the page is reloaded

I'll update the template to load this query only in demo mode then.